pkgs/tools/admin/zbctl/default.nix

```
@@ -0,0 +1,45 @@
+{ lib, stdenvNoCC, fetchurl, coreutils }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "zbctl";
+  version = "8.0.6";
+
+  dontUnpack = true;
+  dontConfigure = true;
+  dontBuild = true;
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/bin
+    cp $src $out/bin/zbctl
+    chmod +x $out/bin/zbctl
+    runHook postInstall
+  '';
+
+  src = if stdenvNoCC.hostPlatform.system == "x86_64-darwin" then fetchurl {
+    url = "https://github.com/camunda/zeebe/releases/download/${version}/zbctl.darwin";
+    sha256 = "17hfjrcr6lmw91jq24nbw5yz61x6larmx39lyfj6pwlz0710y13p";
+  } else if stdenvNoCC.hostPlatform.system == "x86_64-linux" then fetchurl {
+    url = "https://github.com/camunda/zeebe/releases/download/${version}/zbctl";
+    sha256 = "1xng11x7wcjvc0vipdrqyn97aa4jlgcp7g9aw4d36fw0xp9p47kp";
+  } else throw "Unsupported platform";
```

Looks like I missed this before 😅 

```suggestion
  } else throw "Unsupported platform ${stdenvNoCC.hostPlatform.system}";
```